package.json

```diff
@@ -36,94 +36,95 @@
   },
   "dependencies": {
```

Comment:
Can you roll back the updates to the `dependencies`?

I get the idea here. However, we're eventually going to use Greenkeeper to keep our dependencies up to date, but I'd like to not touch anything until we're ready for that.

`devDependencies` updates are fine as long as everything works, but wholesale bumps of production dependencies in a tooling PR makes me nervous.